integrationtests/server-integration/wildfly-modules/src/test/resources/arquillian.xml

```diff
@@ -11,17 +11,16 @@
             <configuration>
                 <property name="jbossHome">${basedir}/target/node1</property>
                 <property name="javaHome">${server.jvm}</property>
+                <!-- To debug the Arquillian managed application server: Add the following javaVmArguments
+                -agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=y
+                -->
                 <property name="javaVmArguments">
                     ${server.jvm.args}
+                    -agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=y
```

Comment:
I think `suspend=y` might be why the CI tests are failing :slightly_smiling_face:

Reply:
Removed